resolwe/flow/static/flow/fieldSchema.json

```diff
@@ -249,6 +250,53 @@
             }
         },
 
+        "_single_data": {
+            "type": "object",
+            "required": ["name", "label", "type"],
+            "additionalProperties": false,
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "pattern": "^[a-zA-Z][a-zA-Z0-9_]*$"
+                },
+                "label": {
+                    "type": "string"
+                },
+                "type": {
+                    "type": "string",
+                    "pattern": "^data:([a-z0-9]+:)*$"
+                },
+                "description": {
+                    "type": "string"
+                },
+                "required": {
+                    "type": "boolean"
+                },
+                "disabled": {
+                    "type": ["string", "boolean"]
+                },
+                "hidden": {
+                    "type": ["string", "boolean"]
+                },
+                "relations": {
+                    "type": "object",
+                    "required": ["type", "npartitions"],
+                    "additionalProperties": false,
+                    "properties": {
+                        "type": {
+                            "type": "string"
+                        },
+                        "npartitions": {
+                            "anyOf": [
+                                {"type": "integer"},
+                                {"type": "string", "patteren": "^none|any$"}
```

Comment:
I propose we initially support defining a number 1 and not defining the property at all (not required).